packages/yup/src/helpers/validate-yup.ts

```diff
@@ -0,0 +1,46 @@
+import type {
+  ErrorRecord,
+  FormKey,
+  FormValidateCallback,
+} from 'ember-headless-form';
+import { assert } from '@ember/debug';
+
+import type { ObjectSchema, ValidationError } from 'yup';
+
+export default function validateChangeset<DATA extends object>(
```

Comment:
Late to this...but how come this is called `validateChangeset` instead of `validateYup`?

Although the name probably doesn't matter since it's a default export?

Reply:
the name does matter for debugging!

Reply:
Oh, copypasta! 🙈 Will fix. Thanks for catching this!